### PR TITLE
FIX: don't autorotate xpdf images

### PR DESCRIPTION
**Pull request by jklymak**
## PR Summary

As per #10290, ghostscript auto-rotates during `ps2pdf`, so if a figure has a *lot* of roated text it gets unexpectedly rotated if the user has `rcParams['ps.usedistiller'] = "xpdf"` set.

This passes the flag "-dAutoRotatePages=false" to `ps2pdf`.

See also 

https://www.ghostscript.com/doc/current/Ps2pdf.htm

## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is PEP 8 compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: lib/matplotlib/backends/backend_ps.py [hunk 1]
```diff
@@ -1557,12 +1557,14 @@ def xpdf_distill(tmpfile, eps=False, ptype='letter', bbox=None, rotated=False):
         command = [str("ps2pdf"), "-dAutoFilterColorImages#false",
                    "-dAutoFilterGrayImages#false",
                    "-sGrayImageFilter#FlateEncode",
+                   "-dAutoRotatePages=false",
```
Comment (anntzer) [inline lib/matplotlib/backends/backend_ps.py]: needs a hash to make windows happy... (see above too)
unfortunately we're not currently testing this codepath on windows CI

File: lib/matplotlib/backends/backend_ps.py [hunk 2]
```diff
@@ -1544,7 +1544,7 @@ def xpdf_distill(tmpfile, eps=False, ptype='letter', bbox=None, rotated=False):
     """
     pdffile = tmpfile + '.pdf'
     psfile = tmpfile + '.ps'
-
+
```
Comment (anntzer) [inline lib/matplotlib/backends/backend_ps.py]: no trailing spaces

File: lib/matplotlib/backends/backend_ps.py (hunk 1 above)
Comment (jklymak) [reply lib/matplotlib/backends/backend_ps.py]: ooops!

**Comment by jklymak** [comment]
Closing in lieu of #10295

**Pull request closed without merging**